### ISSUE TITLE
Invalid JSON from CS Kubernetes resource causes crash in `terraform plan`

### ISSUE THREAD

**Issue by mehemken** [alibaba/terraform-provider#706]
Hello,

I built an `alicloud_cs_kubernetes` resource which failed due to an invalid string input. Ok that is fine. I can fix an invalid string input.

<details>
<summary> Invalid string error</summary>
<pre>
<code>
Error: Error applying plan:
1 error(s) occurred:
* module.main_cluster.alicloud_cs_kubernetes.kub_cluster: 1 error(s) occurred:
* alicloud_cs_kubernetes.kub_cluster: Waitting for kubernetes cluster "running" got an error: &errors.errorString{s:"Waitting for cluster cde127aa91e1343f481dbfe84b6bbfda5 running failed. Looking the specified reason in the web console."}
</code>
</pre>
</details>

## However

When I tried to `terraform destroy` it, I got the error below. I get the same error when I do a `terraform plan`. I cannot move forward with this project due to this error. **How can I fix this?**

```
Error: Error refreshing state: 1 error(s) occurred:

* module.main_cluster.alicloud_cs_kubernetes.kub_cluster: 1 error(s) occurred:

* module.main_cluster.alicloud_cs_kubernetes.kub_cluster: alicloud_cs_kubernetes.kub_cluster: unexpected end of JSON input
```

Currently there is a cluster in my Aliyun account in a failed state. I am hesitant to delete it because it might break my Terraform state.

## Version info
```
Terraform v0.11.8
+ provider.alicloud v1.17.0
```

**Comment by mehemken** [comment]
A workaround was:

```
terraform state rm <resource>
```

and then deleting it manually in the web UI.

**Comment by mehemken** [comment]
Hello @xh4n3 thank you for your message. This happened 8 days ago and I no longer have that exact state file. However I can reproduce the situation. In fact, I can use the same failed cluster that I will use for issue #723 .

1. The cluster will try to create the NAT gateway
1. It will fail to build the NAT gateway
1. The cluster will complete building in a failed state
1. Terraform will exhibit the behavior described in this issue

I will paste the error soon.

**Comment by mehemken** [comment]
Ok the output relevant here has been pasted in issue #723. The output is [in this comment](https://github.com/alibaba/terraform-provider/issues/723#issuecomment-429109865)

**Comment by xiaozhu36** [comment]
Closed by #723
